packages/builders/scripts/fast-usdc/add-operators.build.js

```diff
@@ -0,0 +1,83 @@
+// @ts-check
+import { makeHelpers } from '@agoric/deploy-script-support';
+import { getManifestForAddOperators } from '@agoric/fast-usdc/src/add-operators.core.js';
+import { toExternalConfig } from '@agoric/fast-usdc/src/utils/config-marshal.js';
+import { configurations } from '@agoric/fast-usdc/src/utils/deploy-config.js';
+import { Far } from '@endo/far';
+import { parseArgs } from 'node:util';
+
+/**
+ * @import {CoreEvalBuilder, DeployScriptFunction} from '@agoric/deploy-script-support/src/externalTypes.js';
+ * @import {ParseArgsConfig} from 'node:util';
+ * @import {FastUSDCConfig, FeedPolicy} from '@agoric/fast-usdc/src/types.js';
+ * @import {FastUSDCOpts} from './init-fast-usdc.js';
+ */
+
+const { keys } = Object;
+
+/** @type {ParseArgsConfig['options']} */
+const options = {
+  net: { type: 'string' },
+  oracle: { type: 'string', multiple: true },
+};
+const oraclesUsage = 'use --oracle name:address ...';
+
+const crossVatContext = /** @type {const} */ ({
+  /** @type {Brand<'nat'>} */
+  USDC: Far('USDC Brand'),
+});
+
+/** @type {CoreEvalBuilder} */
+export const defaultProposalBuilder = async (
+  powers,
+  /** @type {FastUSDCConfig} */ config,
+) => {
+  return harden({
+    sourceSpec: '@agoric/fast-usdc/src/add-operators.core.js',
+    /** @type {[string, Parameters<typeof getManifestForAddOperators>[1]]} */
+    getManifestCall: [
+      getManifestForAddOperators.name,
+      {
+        options: toExternalConfig(config, crossVatContext),
+      },
+    ],
+  });
+};
+
+/** @type {DeployScriptFunction} */
+export default async (homeP, endowments) => {
+  const { writeCoreEval } = await makeHelpers(homeP, endowments);
+  const { scriptArgs } = endowments;
+
+  /** @type {{ values: FastUSDCOpts }} */
+  // @ts-expect-error ensured by options
+  const {
+    values: { oracle: oracleArgs, net },
+  } = parseArgs({ args: scriptArgs, options });
+
+  const parseOracleArgs = () => {
+    if (net) {
+      if (!(net in configurations)) {
+        throw Error(`${net} not in ${keys(configurations)}`);
+      }
+      return configurations[net].oracles;
+    }
+    if (!oracleArgs) throw Error(oraclesUsage);
+    return Object.fromEntries(
+      oracleArgs.map(arg => {
+        const result = arg.match(/(?<name>[^:]+):(?<address>.+)/);
+        if (!(result && result.groups)) throw Error(oraclesUsage);
+        const { name, address } = result.groups;
+        return [name, address];
+      }),
+    );
+  };
+
+  const config = harden({
+    oracles: parseOracleArgs(),
+  });
+
+  await writeCoreEval('add-operators', utils =>
+    defaultProposalBuilder(utils, config),
+  );
+};
```

Comment:
80 lines of almost completely boilerplate code. This gets boring.

The platform is fixed. The platform is fixed. The platform is fixed...

Reply:
yeah, I felt like I was fighting the APIs a bit here. another day.